### PR TITLE
Add @available macro to suppress warning in Xcode 9

### PR DESCRIPTION
**Pull request by PowerPan**
Remove Compiler Warnings in Xcode 9.0

### CODE REVIEW

**Comment by honkmaster** [comment]
Thanks. However, the @available macro is not available in Xcode 8, which is still used by some developers.

**Comment by honkmaster** [comment]
We will leave this issue open for some time and merge it later on.

File: SVProgressHUD/SVProgressHUD.m [hunk 1]
```diff
@@ -249,7 +249,11 @@ + (void)showInfoWithStatus:(NSString*)status {
     
 #if TARGET_OS_IOS && __IPHONE_OS_VERSION_MAX_ALLOWED >= 100000
     dispatch_async(dispatch_get_main_queue(), ^{
-        [[self sharedView].hapticGenerator notificationOccurred:UINotificationFeedbackTypeWarning];
+        if (@available(iOS 10_0, *)) {
```
Comment (juliensaad) [inline SVProgressHUD/SVProgressHUD.m]: I've never seen `@available` used with an underscore for the version number. Would you mind switching this to `if (@available(iOS 10.0, *)) {`?

File: SVProgressHUD/SVProgressHUD.m [hunk 2]
```diff
@@ -249,7 +249,11 @@ + (void)showInfoWithStatus:(NSString*)status {
...
+            [[self sharedView].hapticGenerator notificationOccurred:UINotificationFeedbackTypeWarning];
+        } else {
```
Comment (juliensaad) [inline SVProgressHUD/SVProgressHUD.m]: I'd remove the `else` cases since no code is inside the scope.

**Comment by juliensaad** [comment]
Another small change: I'd appreciate if you could clarify the commit message. Perhaps something like: "Add @available macro to suppress warning in Xcode 9" (exactly like your PR's title).

**Comment by honkmaster** [comment]
Fixed with #870.

**Pull request closed without merging**